src/ssl.c

```diff
@@ -55992,36 +55994,438 @@ unsigned int wolfSSL_CONF_CTX_set_flags(WOLFSSL_CONF_CTX* cctx,
     return cctx->flags;
 }
 
-#ifndef NO_WOLFSSL_STUB
 /**
  * finish configuration command operation
  * @param cctx  a pointer to WOLFSSL_CONF_CTX structure to be set
- * @return WOLFSSL_SUCCESS on success, 
- *     otherwise WOLFSSL_FAILURE (stub currently returns WOLFSSL_FAILURE always)
+ * @return WOLFSSL_SUCCESS on success
  */
 int wolfSSL_CONF_CTX_finish(WOLFSSL_CONF_CTX* cctx)
 {
-    WOLFSSL_STUB("wolfSSL_CONF_CTX_finish");
     (void)cctx;
-    return WOLFSSL_FAILURE;
+    return WOLFSSL_SUCCESS;
+}
+/* 
+ * This comment attempts to describe following definitions ans static functions
+ * that are used for wolfSSL_CONF_cmd() to handle command.
+ * 
+ * The following deinitions use for a part of conf_cmds_tbl[] contents.
```

Comment:
Spelling - "deinitions" should be "definitions"

Reply:
Fixed